tests/Aspire.NATS.Net.Tests/NatsContainerFixture.cs

```diff
@@ -0,0 +1,35 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Testcontainers.Nats;
+using Xunit;
+
+namespace Aspire.NATS.Net.Tests;
+
+public sealed class NatsContainerFixture : IAsyncLifetime
+{
+    public NatsContainer? Container { get; private set; }
+
+    public string GetConnectionString() => Container?.GetConnectionString() ??
+        throw new InvalidOperationException("The test container was not initialized.");
+
+    public async Task InitializeAsync()
+    {
+        if (RequiresDockerTheoryAttribute.IsSupported)
+        {
+            Container = new NatsBuilder()
+                .WithImage("nats:2")
```

Comment:
With https://github.com/dotnet/aspire/pull/2856 we are putting these versions in a single place. Can you use this as well? That way we are always testing with the image version we are using in Hosting.